kroxylicious-operator/src/main/resources/META-INF/fabric8/kafkaprotocolfilters.filter.kroxylicious.io-v1.yml

```diff
@@ -95,6 +98,8 @@ spec:
                           The value should be a CamelCase string. 
                           This field may not be empty.
                         type: string
+                        default: ""
```

Comment:
The description says it may not be empty, not sure there's a sane default.

Reply:
Hmm good point, that is a bit strange. To quote KEP-1623: 

> `reason` is required and must not be empty. The actor setting the value should always describe why the condition is the way it is, even if that value is "unknown unknowns". No other actor has the information to make a better choice.

Will remove the default, its going to make the tests interesting.